cypress/e2e/cloud/users.test.ts

```diff
@@ -8,7 +8,7 @@ describe('Users Page', () => {
           cy.quartzProvision({
             hasUsers: true,
           }).then(() => {
-            cy.visit(`/orgs/${id}/users`)
+            cy.visit(`/orgs/${id}/members`)
```

Comment:
do you think this file should be renamed to `members.test.ts`?

Reply:
yes it would make sense to name it `memebers.test.ts` but doing so will require us to rename many files inside `Src/Users` folder, wouldn't it? I fell into this rabbit hole of having to make a lot of changes to many files yesteday because all files linked to `/user` have `user` in the file name

Reply:
i think this one is your call, bucky. It can be done but it will affect many many files. Same thing with `about.test.ts` unfortunately.

Reply:
maybe I should write up a ticket to update the folders I mentioned above?

Reply:
great answer! How about you write up a ticket to change all the instances, but we don't worry about it now. And you and I both file this lesson away as a reminder that figuring out the naming of things early on is way easier than renaming things afterwards.

Reply:
[here](https://github.com/influxdata/ui/issues/5428) is the ticket just for reference